File: CSharp/35_Search_Insert_Position.cs
public class Solution {
    public int SearchInsert(int[] nums, int target) {
        
        var index = 0;
        
        for(var i = 0; i < nums.Length;i++){
            if(target == nums[i]){return i;}
        }

        for(var y = 0; y < nums.Length;y++){
            if(target > nums[y]){index++;}
        }

        return index;
    }
}

File: Python/35_Search_Insert_Position.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:        
        index = 0
    
        for i in range(len(nums)):
            if target == nums[i] :
                return i

        for y in range(len(nums)):
            if target > nums[y] :
                index+=1
        
        return index

        
